README.md

```diff
@@ -464,7 +446,7 @@ maintaining the Node.js project.
 
 ### Release Team
 
-Node.js releases are signed with one of the following GPG keys:
+Ayo.js releases were signed with one of the following GPG keys:
```

Comment:
They never were. I think we should get rid of this section or say "Ayo.js is forked from Node.js, and their releases were signed with..."